tests/test_dsl.py

```diff
@@ -255,6 +256,30 @@ def create(self) -> tosca.artifacts.Root:
     }
 
 
+from unittest.mock import MagicMock
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        (dict(file="foo.yaml"), "from .foo import *"),
+        (dict(file="../foo.yaml"), "from ..foo import *"),
+        (dict(file="../../foo.yaml"), "from ...foo import *"),
+        (dict(repository="unfurl", file="foo.yaml"), "from unfurl.foo import *"),
+        (dict(repository="repo", file="foo.yaml", namespace_prefix="tosca.ns", uri_prefix="tosca.ns"), "from repo import foo as tosca.ns"),
+        (dict(repository="repo", file="foo.yaml", namespace_prefix="ns", uri_prefix="ns"),             "from repo import foo as ns"),
+        (dict(repository="repo", file="foo.yaml", namespace_uri="ns", uri_prefix="ns"),                "from repo import foo as ns"),
```

Comment:
you can delete `uri_prefix` references -- that was just a dumb error on my part, i got the name wrong